app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -75,13 +78,17 @@ class HomeFragment : Fragment() {
         val homeTiles = mutableListOf<HomeTile>()
         val inputAsString = context.assets.open(HOME_TILES_JSON_PATH).bufferedReader().use { it.readText() }
         val jsonArray = JSONObject(inputAsString).getJSONArray(HOME_TILES_JSON_KEY)
+        val userDetails = context.getSharedPreferences(HOME_TILES_PREFS, MODE_PRIVATE)
```

Comment:
nit: rename homeTilesPrefs

Reply:
Side note: Another way to do shared prefs without using a separate file is just to call `PreferenceManager.getDefaultSharedPreferences()`. That way you don't need to make a new "shared preferences" file. However, the tradeoff is that when you load SharedPreferences, you need to load the whole file - so if it's really big, this could slow down your app.

In the case of home tiles, where it's possible that people will want to have a ton of custom tiles, it's might be better to use a separate file - but at the same time, that's the first screen that gets loaded, so it might not matter. But better safe than sorry! so let's keep this separate pref file.